### PR TITLE
Added ovh_refund and ovh_refund_details tables

### PR DESCRIPTION
**Pull request by jdenoy**
Added ovh_refund and ovh_refund_details tables

API paths : 

- /me/refund
- /me/refund/{refundId}
- /me/refund/{refundId}/details/{refundDetailId}

### CODE REVIEW

File: docs/tables/ovh_refund_detail.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+```sql
+select
+  *
+from
+  ovh_refund_detail
+where
+  refund_id = 'AFRxxxxxxx';
+```
+
+### Get one detail of one refund
```
Comment (francois2metz) [inline docs/tables/ovh_refund_detail.md:20]:
```suggestion
### Get detail of one refund
```

File: docs/tables/ovh_refund.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+select
+  id,
+  date,
+  original_bill_id,
+  price_with_tax
+from
+  ovh_refund;
+```
+
+### Get a bill
```
Comment (francois2metz) [inline docs/tables/ovh_refund.md:21]:
```suggestion
### Get a refund
```

File: ovh/table_ovh_refund.go [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+// type RefundPrice struct {
+// 	Value        float64 `json:"value"`
+// 	CurrencyCode string  `json:"currencyCode"`
+// }
```
Comment (francois2metz) [inline ovh/table_ovh_refund.go:16]: Please remove these lines

**Review by francois2metz — COMMENTED** [review]
Thanks for the PR!

File: ovh/table_ovh_refund.go [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+				Hydrate:     getRefundInfo,
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "Date of the refund.",
+			},
+			{
+				Name:        "url",
+				Hydrate:     getRefundInfo,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Url"),
+				Description: "URL to download the bill.",
```
Comment (francois2metz) [inline ovh/table_ovh_refund.go:57]: Is it the url of the bill or the refund?

**Comment by jdenoy** [comment]
Lots of description mistakes on my part. 😞 
I've gone over the complete code and doc to make sure I hadn't missed anything else.
It should be good. 🤞 

I'm currently working on the "kube" tables. Will try my best to check all the descriptions. 🚀

**Comment by francois2metz** [comment]
> Lots of description mistakes on my part. 😞 I've gone over the complete code and doc to make sure I hadn't missed anything else. It should be good. 🤞

Thanks! This is something that take the longuest time in my experience.

> 
> I'm currently working on the "kube" tables. Will try my best to check all the descriptions. 🚀

:+1:

**Comment by francois2metz** [comment]
I squashed the commits and merged your changes manually. Thanks!

**Pull request closed without merging**